app/components/EventItem/index.js

```diff
@@ -0,0 +1,96 @@
+// @flow
+
+import React from 'react';
+import styles from './styles.css';
+import Pill from 'app/components/Pill';
+import { colorForEvent } from 'app/routes/events/utils';
+import { Link } from 'react-router';
+import { Image } from 'app/components/Image';
+import Time from 'app/components/Time';
+import Tag from 'app/components/Tags/Tag';
+import { Flex } from 'app/components/Layout';
+import type { Event } from 'app/models';
+import moment from 'moment-timezone';
+
+type AttendanceProps = {
+  registrationCount: number,
+  totalCapacity: number,
+  event: Event
+};
+
+const Attendance = ({
+  registrationCount,
+  totalCapacity,
+  event
+}: AttendanceProps) => {
+  const future = moment().isAfter(event.activationTime);
+  return (
+    <Pill style={{ marginLeft: '5px', color: 'black', whiteSpace: 'nowrap' }}>
+      {future
+        ? `${totalCapacity} plasser`
+        : `${registrationCount} / ${totalCapacity}`}
+    </Pill>
+  );
+};
+
+type TimeStampProps = {
+  event: Event,
+  field: string
+};
+
+const TimeStamp = ({ event, field }: TimeStampProps) => {
+  const future = moment().isAfter(event.activationTime);
+  const registration = future
+    ? `Åpner ${moment(event.activationTime).format('ll HH:mm')}`
+    : `Åpent`;
+
+  return (
+    <div className={styles.eventTime}>
+      {registration}
+      <br />
+      Starter kl: <Time time={event.startTime} format="ll HH:mm" />
+    </div>
+  );
+};
+
+type EventItemProps = {
+  event: Event,
+  field: string,
```

Comment:
Instead of plain old string, you can use `$Keys<typeof Event>` here, to make sure the input values are valid according to the Event type. A simple `"activationTime" | "startTime"` eg. could also be a possible solution, if it should be possible to use only a few fields.

ref. https://flow.org/en/docs/types/utilities/#toc-keys

Reply:
Ohh, that's cool